### ISSUE TITLE
[EN][Npc][Drop][Azuremyst Isle] Mob looting bug

### ISSUE THREAD

**Issue by RocketeerRaccoon** [Atlantiss/BugTracker#7451]
For some reason after killing any mob in Azuremyst Isle, your unable to loot anything from the killed mob, I tried killing a different mob other than the moths and it was doing the same. My guess is the looting tables for Azuremyst Isle are not working as of now.
![wowscrnshot_052116_140927](https://cloud.githubusercontent.com/assets/1485170/15448827/a0b7b22a-1f64-11e6-9bb8-8e757002615f.jpg)
![wowscrnshot_052116_141253](https://cloud.githubusercontent.com/assets/1485170/15448828/a0bbdc24-1f64-11e6-8924-e55bf87a9407.jpg)
![wowscrnshot_052116_142527](https://cloud.githubusercontent.com/assets/1485170/15448829/a0c056c8-1f64-11e6-8451-81d4390c0d51.jpg)

**Comment by user1011** [comment]
Please follow the _How to use?_: https://github.com/Atlantiss/BugTracker

**Comment by LordKnightCecilo** [comment]
I can loot from everything (Made a new test Draenei just to make sure it wasn't a new issue, Paladin like yours), you may want to disable UI addons/mods, I notice you have at least one, sometimes they can muck with looting. I know I've had that issue on more than one occasion.

**Comment by RocketeerRaccoon** [comment]
Oh, I would have never thought of that, well now that you mention it, I tried disabling all addons like you said, it works. Then I had a hunch that it was the XLoot addon that was causing the issue and it was definitely that. Turns out I was using an out of date version of XLoot for 4.3.4, I tried downloading the latest r376-release for 4.3.4 from WoW Ace and it works. Thanks for the help LordKnightCecilo. :D
